.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -12,6 +12,7 @@
 
 ## Checks
 
+- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
```

Comment:
every commit or the latest commit?

Reply:
Everty commit should be signed off, see:
![image](https://user-images.githubusercontent.com/5276001/181196011-ad3d5632-ae88-45b9-b363-b05aa556eda5.png)
I also remember once I only signed off the last one but the DCO failed.

Reply:
And a simple test is here: https://github.com/ray-project/ray/pull/27082

Reply:
GOT IT!